keps/sig-network/5495-deprecate-ipvs-mode-in-kube-proxy/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: Deprecate ipvs mode in kube-proxy
+kep-number: 5495
+authors:
+  - "@adrianmoisey"
+owning-sig: sig-network
+participating-sigs: []
+status: provisional
```

Comment:
while it is technically `provisional` at the moment, we basically never actually merge provisional KEPs (at least in SIG Network), so you might as well proactively change this to `implementable`.